app/views/finders/_finder_meta.html.erb

```diff
@@ -0,0 +1,4 @@
+<% if finder.description %>
+  <meta name="description" content="<%= finder.description %>">
+<% end %>
+<link rel="alternate" type="application/json" href="/api/content<%= finder.slug %>">
```

Comment:
This is the `base_path` of the content_item, that always has the leading slash right?

https://github.com/alphagov/finder-frontend/blob/master/app/presenters/finder_presenter.rb#L20